controllers/clusteroperator_test.go

```diff
@@ -281,12 +281,12 @@ func TestUpdateCOStatusDegraded(t *testing.T) {
 				ProvisioningInterface:     "eth0",
 				ProvisioningIP:            "172.30.20.3",
 				ProvisioningNetworkCIDR:   "172.30.20.0/24",
-				ProvisioningDHCPRange:     "172.30.20.11, 172.30.20.101",
+				ProvisioningDHCPRange:     "172.30.20.11,172.30.20.101",
 				ProvisioningOSDownloadURL: "",
 				ProvisioningNetwork:       "Managed",
 			},
 			expectedConditions: []osconfigv1.ClusterOperatorStatusCondition{
-				setStatusCondition(osconfigv1.OperatorDegraded, osconfigv1.ConditionTrue, "InvalidConfiguration", "ProvisioningOSDownloadURL is required but is empty"),
+				setStatusCondition(osconfigv1.OperatorDegraded, osconfigv1.ConditionTrue, "InvalidConfiguration", "provisioningOSDownloadURL is required but is empty\n"),
```

Comment:
Can we avoid newlines at the end of the error messages?

Reply:
Yep.